oap-server/server-alarm-plugin/src/test/resources/alarm-settings.yml

```diff
@@ -121,3 +121,16 @@ feishuHooks:
       secret: dummysecret
     - url: https://open.feishu.cn/open-apis/bot/v2/hook/dummy_token2
       secret:
+
+welinkHooks:
+  textTemplate: "Apache SkyWalking Alarm: \n %s."
+  webhooks:
+      # you may find your own client_id and client_secret in your app, below are dummy, need to change.
+    - client_id: "19950212172337588234567"
+      client_secret: d7bc50ee-1995-0212-870f-64673cb797fe
```

Comment:
```suggestion
      client_secret: dummy_secret_key
```
Like this.

Reply:
ok, but i wonder why cannot use "d7bc50ee-1995-0212-870f-64673cb797fe" here since it's not real.

Reply:
Because when you face end users, people don't know you, all documents could mislead. 
And using a clearly fake ID could have less risk.